src/acquisition/covidcast/csv_importer.py

```diff
@@ -229,8 +229,6 @@ def extract_and_check_row(row, geo_type):
     except ValueError:
       # expected a number, but got a string
       return (None, 'sample_size')
-    if sample_size is not None and sample_size < CsvImporter.MIN_SAMPLE_SIZE:
-      return (None, 'sample_size')
```

Comment:
Probably can also remove the definition of `MIN_SAMPLE_SIZE` at the top of the file